index.js

```diff
@@ -19,6 +19,32 @@ const getWidth = stream => {
 	return columns;
 };
 
+const getHeight = stream => {
+	const {rows} = stream;
+
+	if (!rows) {
+		return 20;
```

Comment:
The convention for the default height is 24.

I would make it a top-level constant `const defaultTerminalHeight = 24;`.